src/HW_models/nrfx_bsim_redef.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 Oticon A/S
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file redefines macros from nrfx that need to be different for simulated devices.
+ */
+
+#ifndef NRFX_BSIM_REDEF_H
+#define NRFX_BSIM_REDEF_H
+
+#include <stddef.h>
+#include <stdbool.h>
```

Comment:
Those inclusions are done in `<drivers/nrfx_common.h>`. No need to repeat them here. Same applies to `nrf_bsim_redef.h`.